bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/upgrade.yaml

```diff
@@ -266,14 +266,18 @@
     namespace: "{{ namespace }}"
   register: astronomer_houston_backend
 
-- name: "Upgrade Astronomer: Helm upgrade (4/5) Update Airflow configurations"
+  - name: Update supported Airflow versions list (look for pod airflow-upgrade-check-first-run to appear after 30 seconds)
```

Comment:
@sjmiller609 do you want to number this step just like the others?

Reply:
Also quotes for the name may be?

Reply:
good call